README.md

```diff
@@ -1 +1,222 @@
+[![Puppet Forge](https://img.shields.io/puppetforge/v/simp/iptables.svg)](https://forge.puppetlabs.com/simp/iptables)
+[![Puppet Forge Downloads](https://img.shields.io/puppetforge/dt/simp/iptables.svg)](https://forge.puppetlabs.com/simp/iptables)
+[![Build Status](https://travis-ci.org/simp/pupmod-simp-iptables.svg)](https://travis-ci.org/simp/pupmod-simp-iptables)
+
+#### Table of Contents
+
+1. [Overview](#overview)
+2. [Module Description - What the module does and why it is useful](#module-description)
+3. [Setup - The basics of getting started with iptables](#setup)
+    * [What iptables affects](#what-iptables-affects)
+    * [Beginning with iptables](#beginning-with-iptables)
+4. [Usage - Configuration options and additional functionality](#usage)
+5. [Reference - An under-the-hood peek at what the module is doing and how](#reference)
+    * [Data Types](#data-types)
+6. [Limitations - OS compatibility, etc.](#limitations)
+7. [Development - Guide for contributing to the module](#development)
+      * [Acceptance Tests - Beaker env variables](#acceptance-tests)
+
+## Overview
+
+This module provides native types for managing the system IPTables and
+IP6Tables as well as convenience defines and general system configuration
+capabilities.
+
+The ability to use this module to automatically shim through to firewalld is
+optionally supported for legacy systems and modules that are working on
+migrating to firewalld support.
+
+## This is a SIMP module
+
+This module is a component of the [System Integrity Management Platform](https://simp-project.com),
+a compliance-management framework built on Puppet.
+
+Most SIMP modules actively take advantage of this module when used within the
+SIMP ecosystem.
+
+## Module Description
+
+The ``iptables`` module manages all IPTables and IP6Tables rules in an atomic
+fashion. All rules are applied only once per puppet agent run during the
+application of the last executed ``iptables`` resource.
+
+Applying the rules in this manner ensures that avoid situations where you have
+a partially applied IPTables rule set during a failure in your run of puppet
+(someone hits ^C, your system runs out of memory, etc...).
+
+The module also takes additional safety measures to attempt to keep your
+firewall rules in a consistent state over time to include:
+
+* Rolling back to the last configuration if the application of the new configuration fails
+* Rolling back to an 'ssh-only' mode if application of all configurations fail
+
+The goal is to remain in a state where you can be sure that your system is
+tightly restricted but also able to be recovered.
+
+Finally, the module works to ensure that services such as OpenStack, Docker,
+VirtualBox, etc... can apply their rules without being affected by this module.
+The module provides mechanisms to preserve these rules as managed by external
+systems based on regular expression matches.
+
+## Setup
+
+### What iptables affects
```

Comment:
* The README.md appears to be a copypasta of the iptables module instead of the simp_firewalld module
* The examples in the README are not the examples in the spec/acceptance tests